interactive_multiselect_printer.go

```diff
@@ -70,6 +76,12 @@ func (p InteractiveMultiselectPrinter) WithMaxHeight(maxHeight int) *Interactive
 	return &p
 }
 
+// WithNoFilter sets the NoFilter option
+func (p InteractiveMultiselectPrinter) WithNoFilter() *InteractiveMultiselectPrinter {
```

Comment:
Could this be changed to `WithFilter`. Users could disable it with `WithFilter(false)` then.
In PTerm we usually just enable options by default with the `WithXxx` methods. For disabling, we use the approach above :)